langchain4j-agentic/src/main/java/dev/langchain4j/agentic/agent/AgentBuilder.java

```diff
@@ -362,4 +369,13 @@ public AgentBuilder<T> afterAgentInvocation(Consumer<AgentResponse> afterListene
         this.afterListener = this.afterListener.andThen(afterListener);
         return this;
     }
+
+    public AgentBuilder<T> keyDefaultValue(String key, Object value) {
```

Comment:
```suggestion
    public AgentBuilder<T> defaultKeyValue(String key, Object value) {
```
nit: reads more naturally

Reply:
Done.